CONTRIBUTING.md

```diff
@@ -21,6 +22,21 @@ This will let us have a brief discussion about the problem and, hopefully, ident
 
 Note:  If you wish to work on something that already exists on our backlog, you can use that work item as your proposal.  
 
+# Identifing the development branch
+
+The master branch does not accept direct pull requests. Each change must be submitted through an appropriate developement branch. Each upcoming MRTK release has its own development branch (ex: may18_dev).
```

Comment:
Is there a setting to prevent people from opening a pull request on a specific branch?

Reply:
not sure. I can look

Reply:
doesn't look like we can set that :\

Reply:
yes. THAT i can do